mono/metadata/boehm-gc.c

```diff
@@ -881,11 +883,40 @@ mono_gc_free_fixed (void* addr)
 	GC_FREE (addr);
 }
 
+#ifdef HEAP_VALIDATION_FREQUENCY
+static int counter = 0;
+static int validate_frequency = 0;
+static UnityHeapVerifierCallback unity_heap_validation_callback;
+
+void
+mono_gc_set_heap_validate_frequency (int freq)
+{
+	if (freq >= 0 && freq != validate_frequency)
```

Comment:
Nit, is there a reason to check if they are not equal?

Reply:
Not really, no. I can remove it.